omnipaxos_core/tests/recovery_test.rs

```diff
@@ -411,10 +376,12 @@ fn check_leader_is_elected(cfg: &TestConfig, leader_future: KFuture<Ballot>) ->
     (leader, follower)
 }
 
-// Propose and check that the first proposals before any node fails are decided
-// returns expected entries and a promise that the leader is elected
-fn check_first_proposals(sys: &TestSystem, cfg: &TestConfig) -> (Vec<Value>, KFuture<Ballot>) {
-    let (ble, px) = sys.ble_paxos_nodes().get(&1).unwrap();
+/// Propose and check that the first proposals before any node fails are decided
+fn check_initial_proposals(sys: &TestSystem, cfg: &TestConfig) -> Vec<Value> {
```

Comment:
Good point, I'll let each test create the `proposals` vec and pass its reference down to any function that needs it since it will be used later e.g checking later proposals.